src/store/ChartSettingStore.js

```diff
@@ -11,7 +11,7 @@ export default class ChartSettingStore {
         this.mainStore = mainStore;
         this.menu = new MenuStore(mainStore, { route: 'setting' });
         this.ChartSettingMenu = this.menu.connect(Menu);
-        reaction(() => mainStore.state.settings, () => {
+        reaction(() => this.context, () => {
```

Comment:
why did you change this?

Reply:
oh, so sorry, perhaps I am trying to test something and forget to clean it